components/system.css

```diff
@@ -18,6 +18,17 @@
   --color-rose-8: #efd6e3;
   --color-rose-9: #faf4f8;
 
+  --color-blue-0: #000089;
+  --color-blue-1: #0102ad;
+  --color-blue-2: #4528b8;
+  --color-blue-3: #6645c2;
+  --color-blue-4: #8261cc;
+  --color-blue-5: #9b7ed6;
+  --color-blue-6: #b39be0;
+  --color-blue-7: #cab9ea;
+  --color-blue-8: #e1d8f3;
+  --color-blue-9: #f7f7fc;
```

Comment:
I find this palette interesting. The lower end feels very blue, but the higher end feels very purple 🤔